sunpy/sun/models.py

```diff
@@ -198,3 +206,48 @@ def differential_rotation(duration: u.s, latitude: u.deg, *, model='howard', fra
         rotation -= 0.9856 * u.deg / u.day * duration
 
     return Longitude(rotation.to(u.deg))
+
+
+
+_MODEL_DATA_DIR = pathlib.Path(__file__).parent.absolute() / "data"
+_AVRETT_LOESER_ECSV = _MODEL_DATA_DIR / "chromosphere_avrett_Loeser_2008_model.ecsv"
+
+def _read_chromosphere_data():
+    """
+    Reads the chromosphere model data from an ECSV file.
+
+    Returns
+    -------
+    QTable
+        An Astropy QTable containing the chromosphere model data.
+
+    """
+    if not _AVRETT_LOESER_ECSV.exists():
+        raise FileNotFoundError(f"File not found: {_AVRETT_LOESER_ECSV}")
+
+    return QTable.read(_AVRETT_LOESER_ECSV, format="ascii.ecsv")
+
+_chromosphere_cache = None
```

Comment:
Same with this, this is too vague, there could be multiple _chromosphere models in the future.